_data/governments.yml

```diff
@@ -410,14 +410,14 @@ U.K. Central:
   - datagovuk
   - decc
   - defra
-  - Department-for-Work-and-Pensions
```

Comment:
I looks like this org still exists, but just hasn't been active

Reply:
Yes, we are looking to merge that into our core dwp org at the moment.

Reply:
Right, I've merged that into our core org now and removed the Department-for-Work-and-Pensions one so we're good to go I think.